sdk/storage/azure-storage-common/CHANGELOG.md

```diff
@@ -1,18 +1,15 @@
 # Release History
 
-## 12.9.0-beta.2 (Unreleased)
+## 12.9.0 (2024-11-12)
```

Comment:
@microzchang, are we doing these releases, or have the plans change?

Reply:
Will ship this release today, thanks.